source/slang/slang-type-system-shared.h

```diff
@@ -19,6 +19,7 @@ namespace Slang
     X(Half)                     \
     X(Float)                    \
     X(Double)                   \
+    X(Char)                     \
```

Comment:
As mentioned earlier, I think we really need to have a discussion of what we need/want from a type like this, and clearly document it.